tsconfig.json

```diff
@@ -12,7 +12,7 @@
       "forceConsistentCasingInFileNames": true,
       "noImplicitReturns": true,
       "noImplicitThis": true,
-      "noImplicitAny": true,
+      "noImplicitAny": false,
```

Comment:
We should leave this check in unless there's a good reason to do so

Reply:
Fixed